arch/platform/simplelink/cc13xx-cc26xx/sensortag/tmp-007-sensor.c

```diff
@@ -256,14 +286,21 @@ read_data(uint16_t *local_tmp, uint16_t *obj_tmp)
   uint8_t status_data[] = { REG_STATUS };
   uint16_t status_value = 0;
 
+  if(i2c_acquire()) {
```

Comment:
Should be `if(!i2c_acquire()) {`

Reply:
Rebased over develop and addressed this.